smoke/helpers.go

```diff
@@ -9,13 +10,34 @@ import (
 	. "github.com/onsi/gomega/gexec"
 )
 
+var DEFAULT_TIMEOUT = 30 * time.Second
+
+func SetBackend(appName string) {
+	config := GetConfig()
+	if config.Backend == "diego" {
+		EnableDiego(appName)
+	} else if config.Backend == "dea" {
+		DisableDiego(appName)
+	}
+}
+
 func EnableDiego(appName string) {
```

Comment:
For consistency, can we change the remaining usage of this method (e.g. https://github.com/cloudfoundry/cf-smoke-tests/pull/14/files#diff-e0aae18d582689f80a196e7f76658f88L49) and replace the invocations with `SetBackend`?

Then we can make this method not exported